inference-engine/src/vpu/graph_transformer/src/stages/strided_slice.cpp

```diff
@@ -4,15 +4,32 @@
 
 #include <vpu/frontend/frontend.hpp>
 
+#include <vpu/stages/post_op_stage.hpp>
+
 #include <memory>
 #include <vector>
 
-#include <vpu/stages/post_op_stage.hpp>
-
 namespace vpu {
 
 namespace {
 
+int maskStrToInt(std::string mask) {
+    int idx = 0, result = 0;
+
+    for (const auto& character : mask) {
+        if (character == ',') continue;
+
+        if (idx++ > 0) {
+            result <<= 1;
+        }
+        if (character == '1') {
```

Comment:
I think, it'll be better if we assert on `character == 1 || character  == 0`

Reply:
Done.